### DISCUSSION TITLE
Notification card still not showing controls, Android 14

### DISCUSSION THREAD

**Discussion by einzel** [TeamNewPipe/NewPipe#11144] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

I'm not super github-y but I'm having the issues present in these pull requests:
https://github.com/TeamNewPipe/NewPipe/issues/9764
https://github.com/TeamNewPipe/NewPipe/pull/10712

I get the notification popup when playing in NewPipe but it shows no buttons and tapping the card brings me back to the app.

I'm on OriginOS 4 with Android 14. I uninstalled and reinstalled a fresh copy but the problem persists. My understanding from the second pull request was that extra buttons were added for 13+ but they should always be showing by default anyway? I didn't want to start another pull request in case this is a one off issue.

**Comment by opusforlife2** [comment]
Chinese OSs and skins often mess with a lot of stuff, especially notifications. See if you're able to search the internet for solutions here, because Newpipe works perfectly fine for OSs closer to vanilla Android.

**Reply by einzel** [reply]
I had assumed it was a lost cause at first precisely because of this, but then I saw Podcast Addict and BubbleUPNP worked, so that's why I'm digging a little deeper.
